content/spin/v2/cli-reference.md

```diff
@@ -19,6 +19,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/cli-refere
 - [spin deploy](#spin-deploy)
 - [spin doctor](#spin-doctor)
 - [spin help](#spin-help)
+- [spin kube](#spin-kube)
```

Comment:
We don't usually list plugins in the command reference, except for `cloud` which is because it's installed by default.  If we are intending that `spin kube` is "default enough" to list here, should we do the same as for `spin cloud` and also list subcommands?

(For what it's worth, I would be entirely happy to drop the `cloud` subcommands here, and point people to the cloud plugin docs.   But whatever we decide our philosophy to be, it would be good to be consistent with it.)

Reply:
Listing the high-level commands and linking off to the official documentation, like we do with Cloud.